src/main/kotlin/nexters/admin/controller/user/MemberController.kt

```diff
@@ -0,0 +1,49 @@
+package nexters.admin.controller.user
+
+import nexters.admin.common.auth.LoggedInMember
+import nexters.admin.domain.user.member.Member
+import nexters.admin.service.auth.AuthService
+import nexters.admin.service.auth.LoginRequest
+import nexters.admin.service.user.MemberService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import javax.validation.Valid
+
+@RequestMapping("/api/users")
+@RestController
+class MemberController(
+        private val authService: AuthService,
+        private val memberService: MemberService,
+) {
+    // TODO: erase all API JavaDoc
+    /**
+     * POST /api/users/login
+     * RequestBody {
+     *      email: string,
+     *      password: string
+     * }
+     * ResponseBody {
+     *      data: string // valid JWT token
+     * }
+     */
+    @PostMapping("/login")
+    fun login(@RequestBody @Valid request: LoginRequest): ResponseEntity<TokenResponse> {
+        val token = authService.generateMemberToken(request)
+        return ResponseEntity.ok(TokenResponse(token))
+    }
+
+    /**
+     * PUT /api/users/password
+     * Authorization: Bearer valid.jwt.token
+     * RequestBody {
+     *      password: string
+     * }
+     */
+    @PutMapping("/password")
+    fun updatePassword(
+            @LoggedInMember member: Member,
+            @RequestBody @Valid request: UpdatePasswordRequest): ResponseEntity<Void> {
```

Comment:
아래와 같이 `)`에 개행을 넣어주어 아래 그림처럼 컨벤션이 되면 조금 더 이쁠 것 같은데 어떻게 생각하시나요? 개인 취향에 따라 갈릴 것 같네요 ㅎㅎ
```suggestion
    @PutMapping("/password")
    fun updatePassword(
            @LoggedInMember member: Member,
            @RequestBody @Valid request: UpdatePasswordRequest
    ): ResponseEntity<Void> {
```
<img width="491" alt="image" src="https://user-images.githubusercontent.com/57135043/216518117-f2fff0b5-652c-4b6c-b5a7-12d8c4bb2d4a.png">

Reply:
아 네 저도 좋아요. 이미 다른 곳들에서 그렇게 구현되어있고요.

이런 것들 잡기 위해 ktlint 도입해야겠네요.